specification_document-releases/2_1-Metabolomics-Release/mzTab_format_specification_2_1-M_release.adoc

```diff
@@ -509,6 +509,21 @@ MTD description An experiment investigating the effects of Il-6.
 ....
 |============================================================================
 
+[[license]]
+==== license
+
+[cols=",",]
+|============================================================================
+|*Description* |The file’s license for re-use, derivative works and distribution. Typically, only one license SHOULD be reported. Multiple licenses may be reported to distinguish between academic and commercial use.
```

Comment:
I would suggest to add a reference like https://spdx.org/licenses/ to the description to point users to the list of available (and supported) licenses.

Reply:
Since mzTab-M format IMHO should be some sort of data/result exchange and sharing format - maybe we should also mention that ideally a license should be used that allows sharing of the data and then even mention that in the description (... such as CC BY)?